psi4/src/psi4/libmints/wavefunction.h

```diff
@@ -360,6 +360,16 @@ class Wavefunction : public std::enable_shared_from_this<Wavefunction>
 
     void set_doccpi(const Dimension& doccpi);
     void set_soccpi(const Dimension& soccpi);
```

Comment:
Erase `set_doccpi` & `set_soccpi`, I think, as they're never used as-is (and have no implementation?) and are replaced by your new fns.

Reply:
Done! (You're right-- it seems like they didn't have any implementation.) Thank you!